core/arch/arm/plat-ls/conf.mk

```diff
@@ -20,6 +20,10 @@ CFG_BOOT_SYNC_CPU ?= y
 CFG_BOOT_SECONDARY_REQUEST ?= y
 endif
 
+ifeq ($(PLATFORM_FLAVOR),ls1012ardb)
```

Comment:
Could be written as:
```make
ifneq (,$(filter $(PLATFORM_FLAVOR),ls1012ardb ls1043ardb))
include core/arch/arm/cpu/cortex-armv8-0.mk
endif
```
But, you may keep the two `ifeq`s if you prefer.

Reply:
Let's not change it for now, When more platforms will be planned to get added, I will change it.